From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Mon, 14 Jan 2013 19:22:25 +0100
Subject: [PATCH] makefile: don't use TLSFLAGS

They are generated by uname, which is not cross compile safe.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 src/Makefile |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index ac019df..b63e032 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -84,10 +84,10 @@ SWIG_CFLAGS += -Wno-error -Wno-unused-variable -Wno-unused-but-set-variable -Wno
 
 RANLIB=ranlib
 
-ARCH := $(patsubst i%86,i386,$(shell uname -m))
-ifneq (,$(filter i386,$(ARCH)))
-TLSFLAGS += -mno-tls-direct-seg-refs
-endif
+#ARCH := $(patsubst i%86,i386,$(shell uname -m))
+#ifneq (,$(filter i386,$(ARCH)))
+#TLSFLAGS += -mno-tls-direct-seg-refs
+#endif
 
 SWIG = swig -Wall -python -o $(SWIGCOUT) -outdir ./
 
